src/api/app/models/project.rb

```diff
@@ -209,6 +209,18 @@ def subprojects
     Project.where("name like ?", "#{name}:%")
   end
 
+  def siblingprojects
+    p = parent
```

Comment:
Ah I see it is a function, thought it is an association. So how you did it before you were right but with the new query you don't need to loop anymore so you can simply do:

```
def siblingprojects
  Project.where('name REGEXP (?) and name != ?', "^#{parent.try(:name)}:\\w*\\z", name).order(:name)
end
```

Note the ``try``, it would result in ``"^:\\w*\\z"`` regex which result would be empty.

However, this would hit the database. If we don't want to hit the database we could do this:

```
def siblingprojects
  parent = parent.try(:name)
  if parent
    Project.where('name REGEXP (?) and name != ?', "^#{parent}:\\w*\\z", name).order(:name)
  else
    Project.none
  end
end
```